player/player.go

```diff
@@ -247,6 +262,7 @@ func (p *Player) SearchAndAdd(c *gumble.Client, apiKey, query string) (*Track, e
 
 // Clears the tracks from the playlist
 func (p *Player) ClearQueue() {
+	p.SongList = nil
```

Comment:
```suggestion
	p.SongList = make([]string, 0)
```
making it nil would break it